README.md

```diff
@@ -1,63 +1,44 @@
 # Android Analytics for Snowplow
 
-[![actively-maintained]][tracker-classificiation] [![Build Status][gh-actions-image]][gh-actions] [![Coverage Status][coveralls-image]][coveralls] [![Release][release-image]][releases] [![License][license-image]][license]
+[![actively-maintained]][tracker-classificiation]
+[![Build Status][gh-actions-image]][gh-actions]
+[![Coverage Status][coveralls-image]][coveralls]
+[![Release][release-image]][releases]
```

Comment:
`release-image` and `releases` seem to have vanished. I assume it should be:
https://img.shields.io/github/v/release/snowplow/snowplow-android-tracker